supertokens_python/recipe/session/recipe.py

```diff
@@ -226,10 +237,14 @@ async def handle_api_request(
     async def handle_error(
         self, request: BaseRequest, err: SuperTokensError, response: BaseResponse
     ) -> BaseResponse:
+        if isinstance(err, SuperTokensSessionError):
+            for mutator in err.response_mutators:
+                mutator(response=response)  # type: ignore
+
```

Comment:
dont call this here, instead call this in the middleware impl for all of the frameworks before calling the handle error function

Reply:
On think more about it. I think it's much better to do this in session recipe only. Otherwise, recipe's will get tightly coupled with frameworks. Not a good idea.

Reply:
As discussed. I'm leaving it as it is. Just removing `# type: ignore`